lntest/harness.go

```diff
@@ -548,8 +548,10 @@ tryconnect:
 // behave the same as ConnectNodes. If a pending connection request has already
 // been made, the method will block until the two nodes appear in each other's
 // peers list, or until the 15s timeout expires.
-func (n *NetworkHarness) EnsureConnected(ctx context.Context,
-	t *testing.T, a, b *HarnessNode) {
+func (n *NetworkHarness) EnsureConnected(t *testing.T, a, b *HarnessNode) {
+	ctxb := context.Background()
+	ctx, cancel := context.WithTimeout(ctxb, DefaultTimeout)
```

Comment:
There is a child timeout context created below with also the default timeout. But since we're doing that twice, shouldn't we use double the timeout here but get rid of the child context below?

Reply:
Correct, somehow I overlooked it, now fixed!

Reply:
hmmm actually this was fixed in a later commit, but still needed to double the timeout tho